### PR TITLE
Enable script supporting in request

### PR DESCRIPTION
**Pull request by ityaptin**
This feature is needed for resources updating in Telemetry

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: metadata/service/server/cluster.yml [hunk 1]
```diff
@@ -12,3 +12,4 @@ parameters:
       cluster:
         name: elasticsearch
         multicast: true
+      enable_script: true
```
Comment (simonpasquier) [inline metadata/service/server/cluster.yml]: you shouldn't change the previous setting. Please modify the user model instead.

File: elasticsearch/files/elasticsearch.yml [hunk 2]
```diff
@@ -430,3 +430,9 @@ discovery.zen.ping.unicast.hosts: [{% for member in server.cluster.members %}"{{
 #monitor.jvm.gc.old.warn: 10s
 #monitor.jvm.gc.old.info: 5s
 #monitor.jvm.gc.old.debug: 2s
+
+{%- if server.get("enable_script", False) %}
+script:
+  indexed: on
+  inline: on
```
Comment (simonpasquier) [inline elasticsearch/files/elasticsearch.yml]: Can you try to be less permissive and enable only the required extensions? See https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting-security.html.

**Review by simonpasquier — CHANGES_REQUESTED** [review]

File: elasticsearch/files/elasticsearch.yml [hunk 3]
```diff
@@ -430,3 +430,7 @@ discovery.zen.ping.unicast.hosts: [{% for member in server.cluster.members %}"{{
 #monitor.jvm.gc.old.warn: 10s
 #monitor.jvm.gc.old.info: 5s
 #monitor.jvm.gc.old.debug: 2s
+
+{%- if server.script is mapping %}
+script: {{ server.script }}
```
Comment (SwannCroiset) [inline elasticsearch/files/elasticsearch.yml]: you must serialize the dict to yaml : 

`{{ server.script|yaml }}`

**Review by SwannCroiset — CHANGES_REQUESTED** [review]

**Review by simonpasquier — APPROVED** [review]

**Pull request merged**